src/System.Threading.Tasks.Parallel/tests/ParallelForTests.cs

```diff
@@ -16,132 +16,93 @@ public static void RunParallelExceptionTests()
         {
             // ParallelOptions tests
             ParallelOptions options = new ParallelOptions();
-            Assert.Throws<ArgumentOutOfRangeException>(
-               () => { options.MaxDegreeOfParallelism = 0; });
-            Assert.Throws<ArgumentOutOfRangeException>(
-               () => { options.MaxDegreeOfParallelism = -2; });
-
-            // 
+            Assert.Throws<ArgumentOutOfRangeException>(() => options.MaxDegreeOfParallelism = 0);
+            Assert.Throws<ArgumentOutOfRangeException>(() => options.MaxDegreeOfParallelism = -2);
```

Comment:
As long as we're cleaning these up, can we validate the parameter name using the relevant Throws overload?